src/AppInstallerCLICore/Command.cpp

```diff
@@ -226,7 +230,14 @@ namespace AppInstaller::CLI
                 std::ostringstream strstr;
                 if (arg.Alias() != Argument::NoAlias)

```

Comment:
Mildly dangerous check here if you don't change to `std::string_view`.  Comparing pointers only works for the literals.